### PR TITLE
fix: installing_nvidia_gpu_drivers.md white space

### PR DESCRIPTION
**Pull request by SergeCroise**
remove useless white space to avoid crowdin warning

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [x] 1st Pass (Document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Detailed Editorial Review and Peer Review)
- [x] Final approval (Final Review)

### CODE REVIEW

File: docs/desktop/display/installing_nvidia_gpu_drivers.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Nvidia is one of the most popular GPU manufacturers. You can install Nvidia GPU
 
 !!! Note
 
-    The link for pre-installation actions in Nvidia's official guide is broken. To install the Nvidia driver, you must install the necessary utilities and dependencies from their official repository. 
+    The link for pre-installation actions in Nvidia's official guide is broken. To install the Nvidia driver, you must install the necessary utilities and dependencies from their official repository.
```
Comment (SergeCroise) [inline docs/desktop/display/installing_nvidia_gpu_drivers.md:13]: Nvidia or NVIDIA ?
https://www.nvidia.com/fr-fr/

**Review by sspencerwire — APPROVED** [review]

**Comment by sspencerwire** [comment]
Thank you @SergeCroise

File: docs/desktop/display/installing_nvidia_gpu_drivers.md (hunk 1 above)
Comment (sspencerwire) [reply docs/desktop/display/installing_nvidia_gpu_drivers.md:13]: Good question. The rule is that if it is the product name, then all capitals is preferred. If it is the generic term, then lowercase is fine. In this case, it probably should be capitalized for "Nvidia's official guide" but lower case for "Nvidia driver" line. Go ahead and push another PR if you want to fix it. I had to Google the anwer. As far as NVIDIA is concerned, we probably should not only make it capital in the first instance, but add a trademark symbol. [see here for how to do that](https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/#superscript-subscript-and-special-symbols)

Comment (SergeCroise) [reply docs/desktop/display/installing_nvidia_gpu_drivers.md:13]: @sspencerwire  please review !

Comment (SergeCroise) [reply docs/desktop/display/installing_nvidia_gpu_drivers.md:13]: Indeed, during the translation I saw that the problem of spelling NVIDIA/Nvidia is not so simple, see e.g.
https://en.wikipedia.org/wiki/Nvidia
and
https://en.wikipedia.org/wiki/Nvidia_GTC

**Review by sspencerwire — APPROVED** [review]

**Pull request merged**